pkg/skaffold/build/gcb/types.go

```diff
@@ -55,9 +55,12 @@ const (
 	// StatusCancelled  "CANCELLED" - Build was canceled by a user.
 	StatusCancelled = "CANCELLED"
 
-	// RetryDelay is the time to wait in between polling the status of the cloud build
+	// RetryDelay is the time to wait in between writing logs of the cloud build
 	RetryDelay = 1 * time.Second
 
+	// PollingInterval is the time to wait in between fetching grouped cloud build statuses
+	PollingInterval = 10 * time.Second
```

Comment:
I think this is too long.  I suggest that we use a backoff (maybe per-project?) that is reset whenever a new request is lodged.